src/org/usfirst/frc/team2175/properties/JoystickProperties.java

```diff
@@ -54,30 +62,58 @@ public double getDeadbandValue() {
     }
 
     public int getShiftButtonNumber() {
-        return shiftButtonNumber;
+        return Integer.parseInt(shiftButtonInfo[1]);
     }
 
     public int getGearIntakeInNumber() {
-        return gearIntakeInNumber;
+        return Integer.parseInt(gearIntakeInInfo[1]);
     }
 
     public int getGearIntakeOutNumber() {
-        return gearIntakeOutNumber;
+        return Integer.parseInt(gearIntakeOutInfo[1]);
+    }
+
+    public int getRunFeederOutNumber() {
+        return Integer.parseInt(runFeederOutInfo[1]);
+    }
+
+    public int getRunShooterOutNumber() {
+        return Integer.parseInt(runShooterOutInfo[1]);
+    }
+
+    public int getRunShooterInNumber() {
+        return Integer.parseInt(runShooterInInfo[1]);
+    }
+
+    public int getRunFeederInNumber() {
+        return Integer.parseInt(runFeederInInfo[1]);
+    }
+
+    public String getShiftButtonLocation() {
+        return shiftButtonInfo[0];
+    }
+
+    public String getGearIntakeInLocation() {
+        return gearIntakeInInfo[0];
+    }
+
+    public String getGearIntakeOutLocation() {
+        return gearIntakeOutInfo[0];
     }
 
-    public int getRunfeeder() {
-        return runfeeder;
+    public String getRunFeederOutLocation() {
+        return runFeederOutInfo[0];
     }
 
-    public int getRunshooter() {
-        return runshooter;
+    public String getRunShooterOutLocation() {
+        return runShooterOutInfo[0];
     }
 
-    public int getRunShooterIn() {
-        return runshooterin;
+    public String getRunShooterInLocation() {
+        return runShooterInInfo[0];
     }
 
-    public int getRunFeederIn() {
-        return runfeederin;
+    public String getRunFeederInLocation() {
+        return runFeederInInfo[0];
```

Comment:
Doing things this way adds a lot of methods....but we can maybe shrink this a lot using inner classes.

Here's a small example: https://gist.github.com/bvisness/51281c09ca03bd4b1b9c5012bea78ace